projects/httplib2/fuzz_request.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python3
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import httplib2
+import atheris
+import tests
+
+def TestOneInput(input_bytes):
+    http = httplib2.Http()
+    ib = b"HTTP/1.0 200 OK\r\n" + input_bytes
```

Comment:
I don't know what is the good way here. This code will generate invalid http message almost all the time.

Proper semi-correct http message generator is a lot of work, maybe someone did it before?